app/assets/javascripts/views/story_view.js

```diff
@@ -463,14 +466,22 @@ module.exports = FormView.extend({
       new Clipboard('.btn-clipboard');
     }
 
-    const descriptionContainer = this.$('.description-wrapper')[0];
-    if (descriptionContainer) {
+    const description = this.$('.story-description')[0];
```

Comment:
That's temporary while we migrate from backbone to react. But you're right, using attributes is way better than classes in this case.